src/main/java/com/hrznstudio/galacticraft/screen/PlayerInventoryGCScreenHandler.java

```diff
@@ -169,4 +170,41 @@ public int getMaxStackAmount() {
             return Pair.of(PlayerScreenHandler.BLOCK_ATLAS_TEXTURE, new Identifier(Constants.MOD_ID, Constants.SlotSprites.OXYGEN_TANK));
         }
     }
+
+    public ItemStack transferSlot(PlayerEntity player, int index) {
+        ItemStack stack = ItemStack.EMPTY;
+        Slot slotFrom = (Slot)this.slots.get(index);
+        if (slotFrom != null && slotFrom.hasStack()) {
+            ItemStack stackFrom = slotFrom.getStack();
+            stack = stackFrom.copy();
+            if (index >= 0 && index < 8) {
```

Comment:
Index should always be greater than 0 as `slots.get(index)` would throw an exception if it wan't
```suggestion
            if (index < 8) {
```